WordPress/Classes/ViewRelated/Post/PostEditor+Publish.swift

```diff
@@ -336,14 +331,19 @@ extension PostEditor where Self: UIViewController {
             let generator = UINotificationFeedbackGenerator()
             generator.prepare()
 
-            if let error = error {
-                DDLogError("Error publishing post: \(error.localizedDescription)")
-                ActionDispatcher.dispatch(NoticeAction.post(self.uploadFailureNotice(action: action)))
-                generator.notificationOccurred(.error)
-            } else if let uploadedPost = uploadedPost {
+            switch result {
+            case .success(let uploadedPost):
                 self.post = uploadedPost
 
                 generator.notificationOccurred(.success)
+            case .error(let error):
+                guard !self.post.shouldAttemptAutoUpload else {
+                    break
```

Comment:
Do you mean to return here instead of break?

Reply:
In this specific case, we need `break`. I want lines 344-346 to execute, if I use `return` here this will never happen.